### ISSUE TITLE
bug: ion-loading custom content missing from v3

### ISSUE THREAD

**Issue by kensodemann** [ionic-team/ionic#18135]
# Bug

**Ionic version:**

[x] **4.x**

**Describe the Bug**

Provide a way to specify custom content for the `ion-loading` component similar to how the Ionic v3 `content` parameter allowed HTML to be injected into the "loading" component

**Describe Preferred Solution**

Potentially could be handled the way it was in v3, or perhaps better by specifying a template or inject via a slot such as:

```html
<ion-loading>
  <div class="custom-spinner-container">
    <div class="custom-spinner-box"></div>
  </div>
</ion-loading>
```

**Additional Context**
This is being requested by a community member who is trying to achieve the following:

basically, all I want to do is being able to not use the `div.loading-wrapper` inside of the ion-loading component, and instead, pass some HTML (typically a div that I can customize using sass) when I call `loadingController.create()` function

**Comment by brandyscarney** [comment]
Thanks Ken! I believe this is a duplicate of https://github.com/ionic-team/ionic/issues/17520

**Comment by kensodemann** [comment]
Since there is a PR in to fix the docs and it is unclear exactly what the correct mechanism is to provide this functionality, I think a feature request makes more sense than a bug, so I close that one and left this one open.

**Comment by brandyscarney** [comment]
Sounds good! The fix should be similar to how alert works with its `message` property, which is dependent on the innerHTML fix.

**Comment by brandyscarney** [comment]
As this is functionality that existed in v3 that has been removed I'm considering this a bug that it's missing.
